numpy/core/src/common/mmapmodule.c

```diff
@@ -0,0 +1,1586 @@
+/*
+ * Author: Sam Rushing <rushing@nightmare.com>
+ *  
+ * Contributors: ZackerySpytz, Yotam Medini, a-deeb
+ * Modified to support mmap with offset - to map a 'window' of a file
```

Comment:
What is the license of this file? Having author and contributor attribution with no copyright release is IMO problematic, but I am not a lawyer. Where did it come from? Also, do we allow github names as contributors?

Reply:
I think it's a complete copy of the one in cpython

Reply:
The original header in cpython is different. If it is copied from there it I think it should at least maintain the original header, probably with the addition of this sentence from the [CPython LICENSE](https://github.com/python/cpython/blob/3.7/LICENSE#L70) "Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019 Python Software Foundation; All Rights Reserved". Looking around at other files in the repo, that would seem to be enough.